webapp/components/test-file-results.html

```diff
@@ -221,24 +213,18 @@
 
       resultsURL(testRun) {
         if (testRun.revision === 'diff') {
-          return `${testRun.results_url}&path=${encodeURIComponent(this.testFile)}`;
+          return `${testRun.results_url}&path=${this.encodedPath}`;
```

Comment:
Actually, I think this should still be `encodedURIComponent(this.path)`, because we are passing the whole path as the value of a query parameter which means we also need to escape slashes. And `encodedPath` only escapes the last component of the path, leaving all the slashes in intact.